content/guides/models/automations/_index.md

```diff
@@ -3,7 +3,341 @@ menu:
+
+  - W&B strongly recommends that you store secrets in a W&B instance of a cloud secrets manager provided by AWS, GCP, or Azure. Secret managers provided by AWS, GCP, and Azure are configured with advanced security capabilities.
+
+  - W&B does not recommend that you use a Kubernetes cluster as the backend of your secrets store. Consider a Kubernetes cluster only if you are not able to use a W&B instance of a cloud secrets manager (AWS, GCP, or Azure), and you understand how to prevent security vulnerabilities that can occur if you use a cluster.
+{{% /alert %}}
+
+To add a secret:
+
+1. If necessary, generate the sensitive string in the webhook's service. For example, generate an API key or set a password. If necessary, save the sensitive string securely, such as in a password manager.
+1. Log in to W&B and go to the **Settings** page.
+1. In the **Team Secrets** section, click **New secret**.
+1. Using letters, numbers, and `_`, provide a name for the secret.
+1. Paste the sensitive string into the **Secret** field.
+1. Click **Add secret**.
+
+Specify the secrets you want to use for your webhook automation when you configure the webhook. See the [Configure a webhook]({{< relref "#configure-a-webhook" >}}) section for more information. 
+
+{{% alert %}}
+Once you create a secret, you can access that secret in your W&B workflows with `$`.
+{{% /alert %}}
+
+#### Add a webhook
+
+{{% alert %}}
+* Only W&B Admins can configure a webhook for a W&B Team.
+* Before you create a webhook, [create any secrets it needs]({{< relref "#add-a-secret" >}}).
+{{% /alert %}}
+
+This section shows how to configure a webhook's URL, as well as any access tokens and secrets it requires.
+
+1. If the webhook requires any access tokens or other sensitive strings, create a secret in W&B for each sensitive string, and make a note of the secret's name.
+1. Log in to W&B and go to the **Settings** page.
+1. In the **Webhooks** section, click **New webhook**.
+1. Provide a name for the webhook. 
+1. Provide the endpoint URL for the webhook.
+1. If the webhook authenticates using an access token, set **Access token** to the name of the secret that contains the access token. When you configure an automation that uses this webhook, the access token will be available in the `$ACCESS_TOKEN` environment variable, and the HTTP header will have `Authorization: Bearer $ACCESS_TOKEN`.
+1. If the webhook validates its payload using a sensitive string, set **Secret** to the name of the secret that contains the sensitive string.
+1. Click **Test** to test the webhook. Optionally, provide a payload to test. Any payload you specify in this step does not persist, and will need to be specified when you [create the webhook automation]({{< relref "#create-webhook-automation" >}}).
+
+{{% alert %}}
+See the [Troubleshoot your webhook]({{< relref "#troubleshoot-your-webhook" >}}) section to view where the secret and access token are specified in
```

Comment:
my opinion: we should just remove this alert and fix the above to tell the user :)

Reply:
I think I took care of this!